pkg/util/labels.go

```diff
@@ -60,6 +60,32 @@ func GenerateLabelsFromSourceInfo(labels map[string]string, info *git.SourceInfo
 	return labels
 }
 
+// AdjustConfigWithImageLabels examines previously retrieved builder image labels and returns
+// any specified update to the scripts URL and destination settings.  Note, we do
+// not set those fields in the Config object itself, so as to not confuse the s2i build case in
+// which the script install code expects those values to set via the explicit command line argument
+// overrides.  Otherwise, we can inadvertently override scripts from the source repo.
+func AdjustConfigWithImageLabels(cfg *api.Config) (string, string) {
+	scriptsURL, destination := "", ""
+	if cfg.BuilderImageLabels == nil {
+		return "", ""
+	}
+
```

Comment:
```suggestion
	if cfg.BuilderImageLabels == nil {
		return "", ""
	}
	
	scriptsURL, destination := "", ""
```
No need to create the variables before checking to see if we are just going to return two empty strings.